docs/supported_apis/dataframe_supported.rst

```diff
@@ -6,671 +6,40 @@ of an operation that is listed as not implemented, feel free to open an issue on
 `GitHub repository`_, or give a thumbs up to already created issues. Contributions are
 also welcome!
 
-The following table is structured as follows: The first column contains the method name.
-The second column contains link to a description of corresponding pandas method.
-The third column is a flag for whether or not there is an implementation in Modin for
-the method in the left column. ``Y`` stands for yes, ``N`` stands for no, ``P`` stands
-for partial (meaning some parameters may not be supported yet), and ``D`` stands for
-default to pandas.
+The following table is structured as follows: The first column contains the method name,
```

Comment:
It seems to me that this paragraph and above one require rephrasing (and maybe permutation). My concern is about the start of the paragraphs - the following table ... . It is not clear what the following table is.

Reply:
Ok, paragraph was rephrased, please take a look.